static/app/views/issueDetails/eventNavigation.tsx

```diff
@@ -0,0 +1,214 @@
+import styled from '@emotion/styled';
+import omit from 'lodash/omit';
+import scrollToElement from 'scroll-to-element';
+
+import {Button, LinkButton} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {TabList, Tabs} from 'sentry/components/tabs';
+import TimeSince from 'sentry/components/timeSince';
+import {IconChevron} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import {defined} from 'sentry/utils';
+import {getShortEventId} from 'sentry/utils/events';
+import {useLocation} from 'sentry/utils/useLocation';
+import {useNavigate} from 'sentry/utils/useNavigate';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+
+type EventNavigationProps = {
+  event: Event;
+  group: Group;
+};
+
+enum EventNavOptions {
+  RECOMMENDED = 'recommended',
+  LATEST = 'latest',
+  OLDEST = 'oldest',
+}
+
+const EventNavLabels = {
+  [EventNavOptions.RECOMMENDED]: t('Recommended Event'),
+  [EventNavOptions.LATEST]: t('Last Event'),
+  [EventNavOptions.OLDEST]: t('First Event'),
+};
+
+const jumpToSections = [
+  {section: '#event-highlights', label: t('Event Highlights')},
+  {section: '#stacktrace', label: t('Stack Trace')},
+  {section: '#exception', label: t('Exception')},
+  {section: '#breadcrumbs', label: t('Replay & Breadcrumbs')},
+  {section: '#tags', label: t('Tags')},
+  {section: '#context', label: t('Context')},
+  {section: '#hydration-diff', label: t('Hydration Error Diff')},
+  {section: '#user-feedback', label: t('User Feedback')},
+  {section: '#replay', label: t('Replay')},
+];
+
+function handleClick(section: string) {
+  scrollToElement(section);
```

Comment:
I need to look into why we have a library for this, you can do this natively with `scrollIntoView`: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Should be a bit smoother than doing it with JS